app/templates/_macros.html

```diff
@@ -18,6 +18,22 @@
   </div>
 {% endmacro %}
 
+{% macro render_static_tab_bar(tabs, active_tab) %}
```

Comment:
I made "static" variants of the tab bar and tab panel macros to allow us to provide the same kind of UI, but where each tab just links to a different server-side view rather than being powered by Bootstrap's tab plugin.

I did this for the "Match Me" functionality because doing all four database queries for every page view seemed a bit profligate.